Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -103,16 +109,32 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 class pfEsHTMLChunk
 {
     public:
+        struct TextLink_Type {};
+        static constexpr TextLink_Type TextLink{};
+
+        enum Types : uint8_t
+        {
+            kEmpty = 0,
+            kParagraph,
+            kImage,
+            kPageBreak,
+            kFontChange,
+            kMargin,
+            kCover, // Just a placeholder, never actually used after compile time
+            kBook,  // another placeholder
+            kDecal,
+            kMovie,
+            kEditable, // placeholder, ver 3.0
+            kTextLink
```

Comment:
To reduce future diff churn:

```suggestion
            kTextLink,
```